stockflux-securities-master/src/components/Alert.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from "react";
+import "./Alert.css";
+
+const Alert = ({ messages, type }) => {
+  const [messagesArray, setMessagesArray] = useState([]);
```

Comment:
You don't need to use state here at all. You have the `messages` item, which contains the message(s) that you can iterate over when rendering your component with JSX.

Reply:
You said earlier that you thought it should check the type so that the user can enter a single message as a string. If you don't think this is needed then I'll change it.

Reply:
I've removed the state and the if statement now